### PR TITLE
Split hashtag-app into 2 apps, modify actorid, remove mtls workaround

### PR DESCRIPTION
**Pull request by LMWF**
# Description

Split hashtag-app into 2 apps - hashtag-counter and hashtag-actor. It's a bit easier to manage especially because only one of them should have a controller and it seems creating a controller dynamically is not recommended.

Also modify the actor id to <hashtag>_<sentiment> (with the '#' stripped right now, there seems to be a bug here).

The set of hashtags are hardcoded in the first stage of the pipeline, feed-generator, and are names of shapes.

Also remove the mtls workaround now that dapr has been upgraded to 0.6.

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [x] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

File: .github/workflows/hashtag-counter.yml [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# ------------------------------------------------------------
```
Comment (amanbha) [inline .github/workflows/hashtag-counter.yml:1]: Consider merging the 2 yamls if possible as most of the steps are similar

File: hashtag-counter/components/pubsub.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+apiVersion: dapr.io/v1alpha1
+kind: Component
```
Comment (amanbha) [inline hashtag-counter/components/pubsub.yaml:2]: the components dir can be added to gitignore as these files are generated by cli

File: hashtag-actor/components/pubsub.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+apiVersion: dapr.io/v1alpha1
```
Comment (amanbha) [inline hashtag-actor/components/pubsub.yaml:1]: the components dir can be added to gitignore as these files are generated by cli, in test

File: hashtag-counter/Controllers/HashTagController.cs [hunk 4]
```diff
@@ -26,34 +25,39 @@ public class HashTagController : ControllerBase
...
         public async Task<IActionResult> PostMessageBinding([FromBody]SocialMediaMessage message)
         {
-            if (this.configuration[AppSettings.AppType] != AppSettings.HashTagCounter)
-            {
-                return BadRequest(new HTTPResponse($"Illegal request for {this.configuration[AppSettings.AppType]}"));
-            }
+            Console.WriteLine("enter messagebinding");
+
+            Console.WriteLine($"{message.CreationDate}, {message.CorrelationId}, {message.MessageId}, {message.Message}, {message.Sentiment}");
+            this.logger.LogTrace($"{message.CreationDate}, {message.CorrelationId}, {message.MessageId}, {message.Message}, {message.Sentiment}");
```
Comment (youngbupark) [inline hashtag-counter/Controllers/HashTagController.cs:39]: logger doesn't work, does it ?

File: hashtag-actor/Actors/HashTagActor.cs [hunk 5]
```diff
@@ -37,23 +38,23 @@ public HashTagActor(ActorService service, ActorId actorId)
...
 
             try 
             {
-                count = await this.StateManager.GetStateAsync<int>(sentiment);
+                count = await this.StateManager.GetStateAsync<int>(hashtagAndSentiment);
             }
             catch (KeyNotFoundException)
             {
-                this.logger.LogInformation($"{sentiment} does not exist. {sentiment} will be initialized to 0.");
+                Console.WriteLine($"{hashtagAndSentiment} does not exist. {hashtagAndSentiment} will be initialized to 0.");
```
Comment (youngbupark) [inline hashtag-actor/Actors/HashTagActor.cs:51]: if you do not use logger, please remove logger factory in contstructor.

**Comment by youngbupark** [comment]
I think we would need to create solution to collect longhaul-test. Then we can also share the dockerfile for all apps instead of creating separate dockerfile.

File: .github/workflows/hashtag-counter.yml (hunk 1 above)
Comment (LMWF) [reply .github/workflows/hashtag-counter.yml:1]: They're very similar but they may deploy separately, so I'll keep separate unless this changes.

File: hashtag-counter/Controllers/HashTagController.cs (hunk 4 above)
Comment (LMWF) [reply hashtag-counter/Controllers/HashTagController.cs:39]: Yes, I'm finding it doesn't log to console in k8s so it doesn't show up.

**Pull request merged**